lib/matplotlib/widgets.py

```diff
@@ -4199,19 +4199,26 @@ class Lasso(AxesWidget):
         Whether to use blitting for faster drawing (if supported by the
         backend). See the tutorial :ref:`blitting`
         for details.
+    props: dict, optional
+        Lasso line properties. See `matplotlib.lines.Line2D` for valid
+        properties. Default props are:
+            `{'linestyle' : '-', 'color' : 'black', 'lw' : 2}`.
     """
 
     @_api.make_keyword_only("3.7", name="useblit")
-    def __init__(self, ax, xy, callback, useblit=True):
+    def __init__(self, ax, xy, callback, useblit=True, props=None):
         super().__init__(ax)
 
         self.useblit = useblit and self.canvas.supports_blit
         if self.useblit:
             self.background = self.canvas.copy_from_bbox(self.ax.bbox)
 
+        style = {**(props if props is not None else
+                    {'linestyle': '-', 'color': 'black', 'lw': 2})}
+
```

Comment:
```suggestion
        style = {'linestyle': '-', 'color': 'black', 'lw': 2}
        if props is not None:
            style.update(props)
```

This will make sure that you will use the default values for properties not passed.

Reply:
Thanks for the review! added.